examples/with-typescript2/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
Not an expert of TS but do we still need to ignore .js files if we are using the webpack loader?